tests/trainer/test_optimizers.py

```diff
@@ -457,3 +585,14 @@ def test_lr_scheduler_with_no_actual_scheduler_raises(tmpdir):
     trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=True)
     with pytest.raises(MisconfigurationException, match='The lr scheduler dict must have the key "scheduler"'):
         trainer.fit(model)
+
+
+def test_invalid_optimizer_dict_raises(tmpdir):
+    """
+    Test exception when lr_scheduler dict has no scheduler
+    """
+    model = BoringModel()
+    model.configure_optimizers = lambda: [{'optimizer': optim.Adam(model.parameters())}, optim.Adam(model.parameters())]
```

Comment:
```suggestion
    model = BoringModel()
    model.configure_optimizers = lambda: [{'optimizer': optim.Adam(model.parameters())}, optim.Adam(model.parameters())]
```
override using inhertence?